lib/update/system/system.go

```diff
@@ -499,12 +579,23 @@ func updateKubectl(planetPath string, logger logrus.FieldLogger) (err error) {
 	for _, path := range []string{defaults.KubectlBin, defaults.KubectlBinAlternate} {
 		out, err = exec.Command("ln", "-sfT", kubectlPath, path).CombinedOutput()
 		if err == nil {
+			logger.Infof("Updated kubectl symlink: %v -> %v.", path, kubectlPath)
 			break
 		}
-		logger.WithFields(logrus.Fields{
-			logrus.ErrorKey: err,
-			"output":        string(out),
-		}).Warn("Failed to update kubectl symlink.")
+		logger.WithError(err).WithField("output", string(out)).Warn(
```

Comment:
What if I want to add another field - should I chain it with another `WithFeild`? Each `WithField` is also potentially allocating memory.

Reply:
Not necessarily, you can still use `WithFields()`, I just think `WithField()` looks cleaner when you have only 1 field.